components/page-component/PageComponent.vue

```diff
@@ -37,6 +40,7 @@
 
 <script>
 import '~/components/colors/colors.css'
+import AvatarComponent from '~/components/avatar-component/avatarComponent'
```

Comment:
AvatarComponent filename should start with a capital

Reply:
changed